### ISSUE TITLE
ReferenceError: EventTarget is not defined in node<16

### ISSUE THREAD

**Issue by ph-fritsche** [testing-library/user-event#817]
- `@testing-library/user-event` version: 14.0.0-beta.3

Relevant code or config

```javascript
new EventTarget()
```

Problem description:

`EventTarget` does not exist in older node versions.
Importing `user-event` results in a `ReferenceError` because:
https://github.com/testing-library/user-event/blob/2c5d9f1369301539826879f95c9f6093f676e2bb/src/utils/dataTransfer/Clipboard.ts#L34

Suggested solution:

Either
- polyfill
- `user-event` requires a DOM implementation which requires `EventTarget` so any DOM implementation should already ship with a polyfill. We might as well just ask our users to set `EventTarget` globally if they use a DOM implementation that does not already do so in an older node environment.

**Comment by ph-fritsche** [comment]
We won't include a polyfill for now as it is unclear if there are any users out there who use environments without `EventTarget`.

**Comment by make-github-pseudonymous-again** [comment]
Just ran into this today. Running Node 14.18.

**Comment by ph-fritsche** [comment]
@make-github-pseudonymous-again Which DOM implementation are you using? Did you manage to set `global.EventTarget` to the `EventTarget` implementation used by your DOM implementation?

**Comment by make-github-pseudonymous-again** [comment]
I have not setup a particular DOM implementation. This is my first try at `user-event`. Maybe this is my problem?

**Comment by ph-fritsche** [comment]
`user-event` only works on the DOM. This can be a real browser or any other DOM implementation. Most people use `jsdom`. This is also included in `jest`.
Depending on how the environment is set up the `window` is merged with `global` (like in the browser and also in jest-jsdom) or not (like in basic node environment with e.g. jsdom). You basically would need to `global.EventTarget = whateverVarIsHoldingWindow.EventTarget`.

**Comment by make-github-pseudonymous-again** [comment]
OK. I may have had too much enthusiasm when the first render-only tests passed. I probably need to do [what is explained here](https://testing-library.com/docs/react-testing-library/setup#using-without-jest) for the interactive tests to work. Thanks for taking the time to lead me on the path of solution.

**Comment by make-github-pseudonymous-again** [comment]
Mmh. I am running into another problem trying to import `jsdom` in my testing environment

    Warning: Accessing non-existent property 'VERSION' of module exports inside circular dependency

Do you have another DOM implementation to recommend to see if it produces the same error?

**Comment by make-github-pseudonymous-again** [comment]
I managed to make it work using `@testing-library/user-event` v13.5.0 instead of v14.0.0-beta.x. It seems [my testing environment](https://github.com/Meteor-Community-Packages/meteor-mocha) already sets up `global.window` and `global.document` but not `document` or `window` or `global.EventTarget`. Attempting to polyfill only `EventTarget` produces more errors. Attempting to just `import` `jsdom` breaks everything (without even trying to polyfill, I've tried it all the way down to v16.5.1).

I found something that works for now...
